exo.jcr.component.ext/src/main/java/org/exoplatform/services/jcr/ext/common/SessionProvider.java

```diff
@@ -219,9 +219,8 @@ else if (!isSystem)
    public synchronized void close()
    {
 
-      if (closed)
-      {
-         throw new IllegalStateException("Session provider already closed");
+      if (closed) {
```

Comment:
WOuldn't it be helpful to print a warning log in the case the provider is closed ?

Reply:
it will be very annoying because it could happen a lot times

Reply:
Log entry added (for trace log level, we display the full stacktrace as well)